sufia-models/lib/generators/sufia/models/templates/config/sufia.rb

```diff
@@ -95,6 +95,14 @@
   # Default is false
   # config.citations = false
 
+  # Add a collection facet to search results.  Possible values are...
+  #   nil (default) - do not include collection facet
+  #   :user - show for logged in users
+  #   :public - show for everyone (e.g. logged in and non-logged in users)
+  # config.collection_facet = nil
+  # config.collection_facet = :user
```

Comment:
Can you drop this line and the next? I think the documentation provided above is good enough as is, myself.